extensions/amp-a4a/rtc-documentation.md

```diff
@@ -115,6 +118,8 @@ The `macros` attribute is optional, i.e. a vendor could specify a URL that has n
 
 Additionally, macros can be substituted in by the Fast Fetch implementation itself. The URL for a vendor will be expanded with the correct macro values prior to callout. See [URL Macro Substitution](#url-macro-substitution) section below for detailed explanation and example.
 
+The `errorReportingUrl` is optional. The only available macros are ERROR_TYPE and HREF. See [RTC Error Pingback](#rtc-error-pingback) section below for more information on how error reporting works.
```

Comment:
Nit: Maybe say "The `errorReportingUrl` property is optional." or just "`errorReportingUrl` is optional.".

Reply:
done